astroid/rebuilder.py

```diff
@@ -104,13 +104,16 @@ def __init__(
             self._parser_module = parser_module
         self._module = self._parser_module.module
 
-    def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
+    def _get_doc(self, node: T_Doc) -> Tuple[Optional[str], Optional[nodes.Const]]:
+        """Returns the docstring of a node both in str and nodes.Const.
+
+        The constant will only be returned in python 3.8+.
```

Comment:
```suggestion
        The constant will only be returned in python 3.8+ because the improved ast parsers
        gives as additional information about line numbers.
```